lighthouse-core/aggregator/aggregate.js

```diff
@@ -77,6 +77,12 @@ class Aggregate {
       throw new Error(msg);
     }
 
+    // Audit resulted in an error, so doesn't contribute to score.
```

Comment:
I like the combo of displaying "-" in red and showing the message in a similar manner to debugString 👍